frontend/packages/client/src/components/ProposalInformation.js

```diff
@@ -249,13 +249,13 @@ const ProposalInformation = ({
         >
           <p className="mb-5">Information</p>
           <InfoBlock
-            title={'Stategie(s)'}
+            title={strategies.length === 1 ? 'Strategy' : 'Strategies'}
             content={
               <div className="is-flex flex-1" onClick={openStrategyModal}>
                 {strategies.map((st, index) => {
                   return (
                     <div className="pr-2" key={`${st}-${index}`}>
-                      <StrategyIcon className="cursor-pointer" />
+                      {st.name}
                     </div>
```

Comment:
Proposal will be using only one strategy for winner calculation, so maybe instead of using the map we could get the first element and show it instead of mapping 
```
const  [strategy] = strategies ?? [ { name: ''}];
```
Also use `Strategy` word instead of `Strategies` bc for now there will be only one.

<img width="183" alt="Screen Shot 2022-07-13 at 17 37 54" src="https://user-images.githubusercontent.com/7526740/178830042-0de508e7-5ff0-4da7-a591-4573cdf246ed.png">